datahub-web-react/src/app/shared/tags/AddTagsTermsModal.tsx

```diff
@@ -80,14 +79,33 @@ export default function EditTagTermsModal({
     resources,
     type = EntityType.Tag,
     operationType = OperationType.ADD,
+    defaultValues = [],
+    onOkOverride,
 }: EditTagsModalProps) {
+    console.log({ defaultValues });
     const entityRegistry = useEntityRegistry();
     const [inputValue, setInputValue] = useState('');
     const [showCreateModal, setShowCreateModal] = useState(false);
     const [disableAction, setDisableAction] = useState(false);
-    const [urns, setUrns] = useState<string[]>([]);
-    const [selectedTerms, setSelectedTerms] = useState<any[]>([]);
-    const [selectedTags, setSelectedTags] = useState<any[]>([]);
+    const [urns, setUrns] = useState<string[]>(defaultValues.map((defaultValue) => defaultValue.urn));
+    const [selectedTerms, setSelectedTerms] = useState<any[]>(
```

Comment:
minor nit: this is the basically the same function twice that you could reuse for tags/terms